eslintrc.json

```diff
@@ -8,53 +8,16 @@
     "jest": true,
     "node": true
   },
+  "plugins": [
+    "sort-requires"
+  ],
   "rules": {
-    "array-bracket-spacing": [ 2, "never" ],
-    "arrow-spacing": 2,
-    "block-spacing": 2,
-    "comma-dangle": [ 2, "always-multiline" ],
-    "comma-spacing": 2,
-    "comma-style": [ 2, "last" ],
-    "eol-last": 2,
-    "eqeqeq": 2,
-    "indent": [ 2, 2, {
-      "SwitchCase": 1
-    } ],
-    "keyword-spacing": 2,
-    "linebreak-style": 2,
-    "max-len": [ 2, 120, {
-      "ignoreComments": true,
-      "ignoreStrings": true,
-      "ignoreTemplateLiterals": true,
-      "ignoreUrls": true
-    } ],
-    "no-console": 2,
```

Comment:
It's [included in](https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js#L93) `eslint:recommended`. I believe it was redundant in the last config.